sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/Utils.java

```diff
@@ -84,7 +84,7 @@ public synchronized Date deserialize(JsonElement json, Type typeOfT, JsonDeseria
                 localDocument.getPartition(),
                 localDocument.getDocumentId(),
                 localDocument.getETag(),
-                localDocument.getOperationTime(),
+                localDocument.getOperationTime() / 1000L,
```

Comment:
See other comment, looks like the wrong place to do this conversion. On SQLite it has to stay ms.

Reply:
This code does not reach sqlite, my bad.